schemas/fingerprint-server-api-for-sdks.yaml

```diff
@@ -5435,13 +5435,16 @@ components:
         - asn
         - name
         - network
+        - type
```

Comment:
Is it expected that this is newly required property? Won't it break webhooks deserialization in v3 SDKs? Or is it okay since it will only be used for V4 SDK?

Reply:
np!